tests/client/test_client_key_meilisearch.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+from tests import common
+from datetime import datetime
+from meilisearch.errors import MeiliSearchApiError
+
+def test_get_keys_default(client):
+    """Tests if public and private keys have been generated and can be retrieved."""
+    key = client.get_keys()
+    assert isinstance(key, list)
+    assert len(key) == 2
+    assert 'actions' in key[0]
+    assert 'indexes' in key[0]
+    assert key[0]['key'] is not None
+    assert key[1]['key'] is not None
+
+def test_get_key(client, test_key):
+    """Tests if a key can be retrieved."""
+    keys = client.get_keys()
```

Comment:
`keys` doesn't get used so this line can be removed.

Reply:
Oopsy I forgot this one